README.md

```diff
@@ -1 +1,81 @@
-# angular-parsers-formatters-readme
\ No newline at end of file
+# $parsers and $formatters
+
+## Overview
+
+Parsers and formatters allow us to have different values in our view and our model for the same item. Don't be confused, they're really simple!
```

Comment:
Add a one sentence description of the problem that parsers and formatters solve.